packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- The method `getERC20TokenName` has been added to the assets contract controller ([#1127](https://github.com/MetaMask/core/pull/1127))
+  - This method gets the token name from the token contract
+
+### Changed
+- **BREAKING:** The tokens controller now requires `onTokenListStateChange` and `getERC20TokenName` as constructor parameters ([#1127](https://github.com/MetaMask/core/pull/1127))
```

Comment:
We may have another breaking change — it looks like the signature of `addToken` changed as well to take an optional object as the fourth and final parameter instead of `image` and `interactingAddress`: https://github.com/MetaMask/core/pull/1127/files#diff-4de3568278666a6d7a1c5084b38802cc730ba3ce94b7fd90de875d758e462d99R285

Reply:
Good catch! Fixed in 6d279237